core/controllers/dashboard.py

```diff
@@ -175,8 +176,18 @@ def _get_intro_card_color(category):
             feedback_services.get_thread_analytics_multi(
                 exploration_ids_subscribed_to))
 
+        unresolved_answers_mapping = (
+            stats_services.get_exp_wise_unresolved_answers_count_for_default_rule(  # pylint: disable=line-too-long
+                exploration_ids_subscribed_to))
+
         for ind, exploration in enumerate(explorations_list):
```

Comment:
(Sorry for not catching this last time!)

I was a bit confused why `explorations_list` was a list of summary dicts, and not exploration domain objects. I suggest naming this `exp_summary_list` instead.